threadpoolctl.py

```diff
@@ -637,7 +637,16 @@ def set_num_threads(self, num_threads):
         return set_func(num_threads)
 
     def _get_extra_info(self):
-        pass
+        self.threading_layer = self.get_threading_layer()
+
+    def get_threading_layer(self):
+        """Return the threading layer of OpenBLAS"""
+        threading_layer = self._dynlib.openblas_get_parallel()
+        if threading_layer == 2:
+            return "openmp"
```

Comment:
This is not covered by any test but I suppose it would be too much of a hassle to setup a dedicated CI config just for this as AFAIK there is no binary package for OpenBLAS built with openmp.